code/__DEFINES/maths.dm

```diff
@@ -18,6 +18,12 @@
 #define PERCENT(val) (round((val)*100, 0.1))
 #define CLAMP01(x) (clamp(x, 0, 1))
 
+#if DM_VERSION < 513
+#define CLAMP(CLVALUE,CLMIN,CLMAX) ( max( (CLMIN), min((CLVALUE), (CLMAX)) ) )
+#else
+#define CLAMP(CLVALUE,CLMIN,CLMAX) clamp(CLVALUE, CLMIN, CLMAX)
+#endif
+
```

Comment:
And this

Reply:
Yea, remove this `CLAMP` macro. Just use `clamp`